src/Utils/getDropDownItems.js

```diff
@@ -0,0 +1,17 @@
+import { get } from 'lodash';
+
+const EMPTY_DROPDOWN_ITEM = { label: '', value: '' };
+
+export const getDropDownItems = (resource, propName, isRequired) => {
+  const items = get(resource, `dropdown.${propName}`, []);
+
+  if (!items) {
```

Comment:
could you please describe a purpose of this check? I can't find the logic behind this

Reply:
+1, will it work? you have default value `[]` above

Reply:
Yes. It's my fault. It needs to update.

Reply:
fixed